youtube/internal.js

```diff
@@ -0,0 +1,73 @@
+const DEBUG = false;
```

Comment:
Should this come from an outside source? An environment variable, or a querystring?

Reply:
Yeah, it definitely should. It can't be an environment variable because we're not using a packager and this runs in the browser environment. A querystring would be good I think.